data/preset/gallery.yaml

```diff
@@ -0,0 +1,560 @@
+                                        slots:
+                                          sub-filter:
+                                            type: Filter.OtherGroup
+                                            references:
+                                              other: home-CG4
+                                            slots:
+                                              sub-filter:
+                                                type: Filter.OtherGroup
+                                                references:
+                                                  other: home-CG5
+                  - type: ContentGroup.ContentGroup
+                    slots:
+                      arrangement:
+                        <<: *main-arrangement
+                      selection:
+                        <<: *selection-home
+                        id: home-CG7
+                        slots:
+                          filter:
+                            type: Filter.Articles
+                            slots:
+                              sub-filter:
+                                type: Filter.OtherGroup
+                                references:
+                                  other: home-CG1
+                                slots:
+                                  sub-filter:
+                                    type: Filter.OtherGroup
+                                    references:
+                                      other: home-CG2
+                                    slots:
+                                      sub-filter:
+                                        type: Filter.OtherGroup
+                                        references:
+                                          other: home-CG3
+                                        slots:
+                                          sub-filter:
+                                            type: Filter.OtherGroup
+                                            references:
+                                              other: home-CG4
+                                            slots:
+                                              sub-filter:
+                                                type: Filter.OtherGroup
+                                                references:
+                                                  other: home-CG5
+                                                slots:
+                                                  sub-filter:
+                                                    type: Filter.OtherGroup
+                                                    references:
+                                                      other: home-CG6
+
+# FIXME: The repeating ContentGroup structure should be replaced with the
+# infinite scrolling arrangement once it's made (T16360).
+- &set-page
+  type: Layout.TopMenu
+  slots:
+    top-menu:
+      <<: *main-menu
+    content:
+      shortdef: 'Layout.InfiniteScrolling(expand: true)'
```

Comment:
If the amoutn of card we display is fixed (because Constrained uses fixed space plus we have a fixed number of ContentGriups) then we should use Layout.Scrolling. Any particular reason for not doing that?